CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachangelog.com/)
 
 ## [Unreleased]
+### Added
+- Support for Ruby 2.3.0
```

Comment:
testing not support, they are  different things. We are gonna validate this in the next round of testing.